### PR TITLE
Added bool config to YGConfig to configure using web defaults

### PR DESCRIPTION
**Pull request by woehrl01**
Added bool config to YGConfig to configure using web defaults. See #445.

### CODE REVIEW

**Comment by emilsjolander** [comment]
`align-items` also differs, web default is `flex-start`. This will also need to change the logic in YGNodeStyleGetFlexShrink to return `1` even though `flex` shorthand is set as well as return `auto` from YGNodeResolveFlexBasisPtr when `flex` shorthand is set.

**Comment by woehrl01** [comment]
according to https://developer.mozilla.org/de/docs/Web/CSS/align-items it's ```stretch``` for ```align-items```. I already changed to return ```1``` on ```flex-shrink```. 
I changed ```YGNodeResolveFlexBasisPtr``` is this what you meant?

File: yoga/Yoga.h [hunk 1]
```diff
@@ -235,6 +235,10 @@ WIN_EXPORT void YGConfigSetExperimentalFeatureEnabled(const YGConfigRef config,
 WIN_EXPORT bool YGConfigIsExperimentalFeatureEnabled(const YGConfigRef config,
                                                      const YGExperimentalFeature feature);
 
+WIN_EXPORT void YGConfigSetUseWebDefaults(const YGConfigRef config, const bool enabled);
```
Comment (emilsjolander) [inline yoga/Yoga.h]: Can you add a comment here expressing that web defaults is the preferred configuration and non-web defaults is to be considered legacy?

**Comment by emilsjolander** [comment]
👍 Thanks!

**Comment by woehrl01** [comment]
Sounds like a good idea to port the tests to web defaults. But it seems that this could be a bit of work. So I'd suggest creating an issue, and I could have a look if I have a little more spare time.

**Comment by emilsjolander** [comment]
Yeah, I might take a stab at it this week! 👍  Tracking in https://github.com/facebook/yoga/issues/451

**Pull request closed without merging**